Makefile

```diff
@@ -0,0 +1,27 @@
+sources = ep_transition
```

Comment:
A Makefile in a Python project!?  :)

Right now we already have testing here using nose and the .coveragerc file.  It feels like this would replace some of that but I'm not sure.

Reply:
We can remove this if you want. I find `make format` and `make lint` useful :D